### PR TITLE
add: notFoundProduct, switch language, fix cursors

### PR DESCRIPTION
**Pull request by NavI-Rap0**

### CODE REVIEW

**Comment by AndriiZakharenko** [comment]
👀

**Review by AndriiZakharenko — APPROVED** [review]

File: src/modules/ProductDetailsPage/ProductNotFoundPage/ProductNotFoundPage.module.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    display: flex;
+    position: fixed;
+    top: 0;
+    left: 0;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 100vw;
+    background-color: $original_hover_and_bg;
+    background-image: url('/public/img/product-not-found.png');
```
Comment (iyehorova) [inline src/modules/ProductDetailsPage/ProductNotFoundPage/ProductNotFoundPage.module.scss]: please replace `/public/img/product-not-found.png` to `/img/product-not-found.png`

**Review by iyehorova — CHANGES_REQUESTED** [review]
Good job!
Please make sure that the image does not overlap the footer and check that it looks good on a mobile layout
And check the comment below

**Review by AndriiZakharenko — APPROVED** [review]

**Comment by AndriiZakharenko** [comment]
👀

**Review by iyehorova — APPROVED** [review]
👀

**Pull request merged**